src/features/game/expansion/components/Promoting.tsx

```diff
@@ -125,14 +134,14 @@ export const PromotingModal: React.FC<Props> = ({
                 height: `${PIXEL_SCALE * 16}px`,
               }}
             />
-            <p className="text-sm">1 x Dawn Breaker Banner</p>
+            <p className="text-sm">{`1 x Witches' Eve Banner`}</p>
           </div>
           <p className="text-sm">Includes:</p>
           <ul className="list-disc">
             <li className="text-xs ml-4">25% SFL discount on seasonal items</li>
             <li className="text-xs ml-4">Free Seasonal Banner</li>
-            <li className="text-xs ml-4">Seasonal Wearable Airdrop</li>
-            <li className="text-xs ml-4">Access to exclusive cosmetics</li>
+            <li className="text-xs ml-4">1 x Mystery Airdrop</li>
+            <li className="text-xs ml-4">Bonus tickets from chores</li>
```

Comment:
I know it might be included in the announcement, but people will ask incessantly on Discord....

(Recommend) Be explicit in the game what the bonus is.  "+1"?